lib/profile/commands/configure.rb

```diff
@@ -78,14 +81,14 @@ def generate_prefill(question)
           result = process.run(question.default_smart, nil)
           output = result.stdout.chomp
           if !result.success?
-            smart_log.debug("Command '#{question.default_smart}' failed to run: #{result.stderr}")
+            smart_log.debug("Command '#{question.default_smart}' failed to run: #{result.stderr.dump}")
           elsif (!question.validation.has_key?(:format) || output.match(Regexp.new(question.validation.format)))
             prefill ||= output
           else
             smart_log.debug("Command result '#{output}' did not pass validation check for '#{question.text}'")
           end
         end
-        prefill ||= question.default
+        prefill = question.default || ""
```

Comment:
I think this ought to be `prefill ||= question.default || ""`. Once the smart default value has finished, it is immediately overwritten unless we use the `||=` setter.

Reply:
Good catch, not sure why I removed that in the first place